### PR TITLE
introduce oc-applicaiton-menu

### PR DESCRIPTION
**Pull request by scramb**
### Description
Introducing OcLeftSidebar, OcSidebarNavItem, OcSidebarNavSubItem, OcSidebarNavHeader and OcSidebarNavDivider.

### Issues
- refs #94 

### To-Do's
- [ ] Fix text size for nav-items
- [ ] fix appearance in ods e.g. files template
- [ ] ...

### CODE REVIEW

File: src/elements/OcSidebarNavSubItem.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<template>
+  <li :class="[active, 'uk-parent']">
+    <router-link :to="target">
```
Comment (tomneedham) [inline src/elements/OcSidebarNavSubItem.vue]: Had a quick go at wiring this up in Phoenix. We need to be able to push routes, with params. https://github.com/owncloud/phoenix/blob/master/src/components/Menu.vue#L107 Maybe we just pass in a route object including `name` and `params` elements?

Comment (DeepDiver1975) [reply src/elements/OcSidebarNavSubItem.vue]: I guess we need that same for the breadcrumbs ....

**Comment by DeepDiver1975** [comment]
@scramb conflicting

File: src/elements/OcSidebarNavItem.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <li :class="active">
+    <router-link :to="target">
+      <span class="uk-margin-small-right">
+        <oc-icon variation="inverted" :name="icon" />
```
Comment (tomneedham) [inline src/elements/OcSidebarNavItem.vue]: https://getuikit.com/docs/text#vertical-alignment might help to fix alignment of icon - helped for @butonic I think

**Pull request merged**